### DISCUSSION TITLE
British Spelling Transcriptions

### DISCUSSION THREAD

**Discussion by struong-man** [openai/whisper#1471] [Q&A] [answered]
I believe the normaliser favours American spelling. Currently I am using the prompt with a few words using British spelling to enforce British transcriptions but I am wondering if there is a better way?

**Comment by phineas-pta** [comment]
fine tuning

**Reply by struong-man** [reply]
Is fine tuning possible through the API: https://api.openai.com/v1/audio/transcriptions?

**Reply by phineas-pta** [reply]
nope unfortunately, u should had specified api in question

**Reply by ryanheise** [reply, marked as answer]
That API also has its own forum: https://community.openai.com/

This GitHub repository (the one you're chatting to us on) contains the open source project Whisper and a forum for discussions concerning it, while the openai.com website contains paid APIs and its own forum for discussions concerning them.
